src/ui/public/notify/notify.less

```diff
@@ -77,6 +77,14 @@
     }
   }
 
+  .toast-banner {
+    // no flex
+  }
+
+  .toast-message-banner {
+    display: block;
+  }
```

Comment:
It's just nested 1 level deep under the top-level container of this file, `.toaster-container`. If it needs to be un-nested then there are a ton of things in this file that could be un-nested as well, so I'd prefer to leave it alone if that's ok.